src/pytest_plugins/consume/consume.py

```diff
@@ -128,6 +128,12 @@ def pytest_configure(config):  # noqa: D103
         config.test_cases = TestCases.from_stream(sys.stdin)
         return
 
+    if input_source == "latest":  # TODO: align this with release tags
+        input_source = (
+            "https://github.com/ethereum/execution-spec-tests/releases/latest/download/"
+            "fixtures_develop.tar.gz"
+        )
```

Comment:
I was thinking we could add an `--input-release` flag that accepts existing tags, i.e develop, eipxxx.

Currently we have the single confict where we might want to use a folder called "latest/" but it would get overridden by the logic here and get the latest fixture release instead. Although right now I don't see this every causing an issue.

Reply:
I think it's simply cleaner to just be able to specify an `--input-release` flag. We could add it in this PR.

Reply:
Sounds good! Will look into adding this over the next couple days!

Reply:
I added a `--latest` flag for now.

Specifying by tag specifically was too complex if we additionally account for all the `*.tar.gz`'s for full releases like v3.0.0. 

Latest always points to `fixtures_develop.tar.gz`.

Reply:
> I think it's simply cleaner to just be able to specify an `--input-release` flag. We could add it in this PR.

Now I've re-read were you meaning using `--input-release` for urls and `--input` for local fixtures?